rpc/gnmi/mixed-schema.md

```diff
@@ -99,6 +99,24 @@ constrained to the specified origin. Particularly:
   the specified `origin`. To delete contents from multiple origins, a client
   MUST specify multiple paths within the `delete` of the `SetRequest`.
 
+### Special considerations for the use of Origin CLI
+* If a `Set` RPC contains `origin` CLI, the CLI origin MUST be given precedence
+  over the OpenConfig origin.  This precedence is independent of the order in
```

Comment:
@dplore shall we further clarify the order precedence for native/default origins?
Say you have a switch that supports the following origins:
1. `cli`
2. no-origin - meaning default origin that may be openconfig or vendor-native schema
3. `oc`

Do we want to say here that cli origin takes precedence over any other potentially available origins (like sonic may have some orthodox `config-db` origin, for instance) and then the rest of the origins are processed in the order-based fashion?

Reply:
@hellt -- I think (as specified in this doc) "" implies origin `openconfig` I believe. Is your implication that `openconfig` (not `oc`) is different to `""` above?

Reply:
I would like to defer the decision of how to deal with 3 origins (OC, Vendor native yang and CLI)  and keep this PR to only OpenConfig (which is also known as "") and CLI origin.    This essentially means at the moment, that there is not a gNMI specification for how to mix vendor native, openconfig and CLI in the same setRequest.  There are other ways for an operator to get deterministic behavior though, by performing separate setrequests for example.